src/App.js

```diff
@@ -1,10 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import data from './data/dummy.json';
 
 const App = () => {
+  const [response, setResponse] = useState({});
```

Comment:
`response` usually means API response, so this could be confusing.

We could rename it to be clearer, to something like `userAnswers`